scss/_skeletons.scss

```diff
@@ -0,0 +1,60 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
+
+  // Animation
+  &.skeleton-glow {
+    animation: skeleton-glow 2s ease-in-out infinite;
+  }
+}
+
+.skeleton-dash {
+  width: 100%;
+  height: 1em;
+  background-color: currentColor;
+
+  + .skeleton-dash {
+    margin-left: 4px;
```

Comment:
`4px` feels wrong, you'd better use one of our spacers (and probably define a variable for this).

Reply:
It was bugging me too. I forgot about the spacers. Here's a quick test. I'd say `$spacer * .5` looks the best? Agree with the variable too. Let me know what you think.

`margin-left: $spacer;`
<img width="281" alt="Screenshot 2020-10-12 at 11 27 26" src="https://user-images.githubusercontent.com/385551/95730079-0be67f80-0c7e-11eb-842e-e9d6f4908108.png">
`margin-left: $spacer * .5;`
<img width="284" alt="Screenshot 2020-10-12 at 11 27 48" src="https://user-images.githubusercontent.com/385551/95730084-0db04300-0c7e-11eb-9ac6-e26896869c47.png">
`margin-left: $spacer * .25;`
<img width="284" alt="Screenshot 2020-10-12 at 11 28 07" src="https://user-images.githubusercontent.com/385551/95730094-10129d00-0c7e-11eb-9411-63310291c683.png">

Reply:
I'd be for quarter myself based on those screenshots, but will need to play a bit with it to make a decision. BTW:
1. we prefer to use division to divide; see [our spacers](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L235).
2. and I guess a variable would be helpful for this `margin`, something like `$skeleton-spacing-x`.

Reply:
Ok, I've changed the margin to `$spacer / 4` to play with it. 
I hold out the variable until we settle down the implementation, is it ok?